reporting/results.py

```diff
@@ -55,6 +56,10 @@ def __str__(self):
         string += f"Characters: {self.characters}\n"
         return string
 
+    @property
+    def unique_id(self):
+        return f"{self.dataset_name} - {self.example_id}"
```

Comment:
Isn't this nomenclature confusing? Pretty sure I see `unique_id == example_id` in the scheduler.

Reply:
It is. And that is why I added a card in JIRA. Currently:
- `example_id`: index of the sample within the dataset (assigned during generation, internal to the dataset).
- `unique_id`: an ID that is unique across the system.

There's also `TestExample.repetition`, which we actually don't use, but it's displayed in the result filename.